reconcile/jenkins_job_builds_cleaner.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import re
+import time
+
+import reconcile.queries as queries
+
+from reconcile.utils.jenkins_api import JenkinsApi
+
+QONTRACT_INTEGRATION = 'jenkins-job-builds-cleaner'
+
+
+def hours_to_ms(hours):
+    return hours * 60 * 60 * 1000
+
+
+def run(dry_run):
```

Comment:
A matter of personal preference, but I find this function too long and deeply indented, which makes following very hard for my sleep-deprived brain. Can we split it?

Reply:
I've split the method in a9b852510c1192fb86483ea091d8a8e343239cbd